.buildkite/linters.sh

```diff
@@ -49,6 +49,13 @@ if [ "$err" -ne 0 ]; then
     globalErr=$err
 fi
 
+err=0
+./tools/scripts/add_27_bindcall.sh -c &> add_27_bindcall || err=$?
```

Comment:
do you think it makes sense to use bazel for this?

Reply:
Maybe, but insofar as nothing else does...I'm not sure it's worth the trouble.